### PR TITLE
SSL for everything

### PR DESCRIPTION
**Pull request by CoolaxGaming**
Just went through all http links, looked for sites that support ssl, and changed to https. Encryption should always be used, weather in code or in comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,12 +110,12 @@ __Help! Things are not working!__
 [Eclipse]: https://eclipse.org/
 [Gradle]: https://gradle.org/
 [Homepage]: https://spongepowered.org/
-[IntelliJ]: http://www.jetbrains.com/idea/
+[IntelliJ]: https://www.jetbrains.com/idea/
 [Issues]: https://github.com/SpongePowered/Sponge/issues
 [Documentation]: https://docs.spongepowered.org/
-[Java]: http://java.oracle.com/
+[Java]: https://java.oracle.com/
```
Comment (liach) [inline README.md]: No need for this one

Comment (kenzierocks) [inline README.md]: Also, on Chrome the cert is invalid.

Comment (hmksq) [inline README.md]: On Firefox the certificate is invalid too!

File: README.md [hunk 2]
```diff
@@ -110,12 +110,12 @@ __Help! Things are not working!__
...
 [Source]: https://github.com/SpongePowered/Sponge/
-[MIT License]: http://www.tldrlegal.com/license/mit-license
+[MIT License]: https://www.tldrlegal.com/license/mit-license
```
Comment (hmksq) [inline README.md]: The site does not have a valid https certificate too!

Comment (ItsAGeekThing) [inline README.md]: Should be changed to:
`[MIT License]: https://tldrlegal.com/license/mit-license`

**Comment by mcsnetworks** [comment]
You did actually test going to the secure version of these pages before you proposed this change right?

**Comment by hmksq** [comment]
Shouldn't this PR already be closed, after all this PR is created by using the GitHub "edit" button, judging from `CoolaxGaming:patch-1`. And this PR breaks links instead, and there are already conflicts when merging.

**Pull request closed without merging**